server/plugin/command.go

```diff
@@ -320,14 +320,15 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 
 	ctx := context.Background()
 	githubClient := p.githubConnectUser(ctx, userInfo)
-
+	username, _ := p.API.GetUser(args.UserId)
 	owner, repo := parseOwnerAndRepo(parameters[0], p.getBaseURL())
+	eventsSubscribed := "`" + strings.Join(strings.Split(features, ","), "`, `") + "`"
```

Comment:
Alternatively, `features` could become a type alias and the implement the [`Stringer`](https://pkg.go.dev/fmt#Stringer) interface.